tests/e2e/e2e_common.go

```diff
@@ -99,6 +99,7 @@ const (
 	e2eTestPassword                            = "E2E-test-password!23"
 	vsphereCloudProviderConfiguration          = "vsphere-cloud-provider.conf"
 	svOperationTimeout                         = 240 * time.Second
+	envVmdkDiskURL                             = "DISK_URL"
```

Comment:
give better name than DISK_URL
also mention in .md file on how the parameter should look like and how to get it

Reply:
names as DISK_URL_PATH, Added to .md file